Jokes_Twitter_Bot.py

```diff
@@ -0,0 +1,31 @@
+import tweepy
+import joke
+
+# Define the API endpoint to fetch memes
+reddit_api_url = "https://www.reddit.com/r/memes/random.json?limit=1"
+
+# Define a user agent to mimic a web browser
+user_agent = (
+    "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 "
+    "(KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36"
+)
+
+meme_data = joke.fetch_random_meme(reddit_api_url, user_agent)
+meme_url = joke.extract_meme_url(meme_data)

```

Comment:
After extraction it should load into json or something.

Reply:
It's only giving back a link of the meme from Reddit

Reply:
That's why I m asking you to load into json then give it to make tweet, see joke.py how i fetched and loaded the data.

I need an hour to look into it deeply as I'm also busy in maintaining other PRs and query, so by that time try something :)

Reply:
okay sure